cross-modal-search/Dockerfile

```diff
@@ -2,12 +2,16 @@ FROM jinaai/jina:2.0.8-perf
 
 RUN apt-get -y update && apt-get install -y git wget gcc unzip libmagic1
 
-COPY . /workspace
 WORKDIR /workspace
+COPY ./requirements.txt requirements.txt
+
+RUN pip3 install torch==1.9.0+cpu torchvision==0.10.0+cpu torchaudio==0.9.0 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
do we need to install `torchaudio==0.9.0`?

Reply:
Nope, neither is torch vision required. I have removed them both. Please see [2978104](https://github.com/jina-ai/examples/pull/684#:~:text=fix%3A%20address%20comments-,2978014,-Merge%20state) for the change.